src/bot/commands/help.ts

```diff
@@ -1,18 +1,40 @@
 import Discord, { Collection } from 'discord.js';
-
 import ICommand from './ICommand';
+import ConfigService from '../../core/services/config.service'
+import container from '../../inversity.config';
 
+const prefix = container.resolve<ConfigService>(ConfigService).get('BOT_TRIGGER');
 const command: ICommand = {
   name: 'help',
   aliases: ['commands'],
-  description: 'Lists available commands.',
+  example: `\`${prefix}help ping\``,
+  description: 'Lists available commands!',
   async execute(message: Discord.Message, args: string[], prefix: string, commands: Collection<string, ICommand>) {
     const data = [];
 
     if (!args.length) {
       // Get for all commands
-      data.push('Here\'s a list of all my commands:\n');
-      data.push(commands.map(command => command.name).join(', '));
+      data.push('here\'s a list of all my commands:\n');
+
+      const cmds = commands.map(command => command.name);
+      var cmd;
```

Comment:
avoid var at all costs :D. Use `let` instead when it's a value that will change later.